src/gui/iconconfig.cpp

```diff
@@ -0,0 +1,390 @@
+#include <QHash>
+#include <QString>
+
+extern const QHash<QString, QString> iconConfig = {
```

Comment:
Since these consts are supposed to be used in one .cpp file I would put them into .h instead and omit extern declaration (or even put them into corresponding .cpp directly).

Reply:
moved in uithememanager.cpp